code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -77,15 +77,17 @@ GLOBAL_LIST_INIT(hardcoded_gases, list("o2","n2","co2","plasma")) //the main fou
 	moles_visible = 1
 	dangerous = TRUE
 
-/datum/gas/oxygen_agent_b
-	id = "agent_b"
-	specific_heat = 300
-	name = "Oxygen Agent B"
+/datum/gas/brown_gas //This is nitric oxide, but given generic name to avoid confusion with nitrous oxide(N20 vs. NO2)
+	id = "browns"
+	specific_heat = 20
+	name = "Brown Gas"
```

Comment:
You need icons for these.

Reply:
also a better name

Reply:
Do you not see the "add sprites for new gases" checkbox.

Reply:
no

Reply:
@Iamgoofball you must've missed "remove goofmos" too